index.js

```diff
@@ -54,12 +53,66 @@ class OktaStatusError extends Error {
   }
 }
 
+class ConfigurationValidationError extends Error {}
+const findDomainURL = 'https://bit.ly/finding-okta-domain';
+const findAppCredentialsURL = 'https://bit.ly/finding-okta-app-credentials';
+const copyCredentialsMessage = 'You can copy it from the Okta Developer Console ' +
+  'in the details for the Application you created. ' +
+  `Follow these instructions to find it: ${findAppCredentialsURL}`;
+
+const isHttps = new RegExp('^https://');
+const hasDomainAdmin = /-admin.(okta|oktapreview|okta-emea).com/;
```

Comment:
Can you use a consistent syntax for regex handling?

Reply:
Yep, I'm using RegExp now